.github/workflows/vulnz-docker-release.yml

```diff
@@ -0,0 +1,46 @@
+name: docker pr
+
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  IMAGE_FQDN: ghcr.io/jeremylong/open-vulnerability-data-mirror
+  VERSION: ${{ github.ref_name }}
+
+jobs:
+  docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - name: Run build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: ./gradlew -x test -Pversion=${{ env.VERSION }} vulnz:build
+      - name: Build docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: vulnz/
+          platforms: linux/amd64
+          push: true
+          tags: |
+            ${{ env.IMAGE_FQDN }}:${{ env.VERSION }}
```

Comment:
```suggestion
            ${{ env.IMAGE_FQDN }}:${{ env.VERSION }}
            ${{ env.IMAGE_FQDN }}:latest
            ghcr.io/${{ env.IMAGE_FQDN }}:${{ env.VERSION }}
            ghcr.io/${{ env.IMAGE_FQDN }}:latest
```

Reply:
Did it a bit differently

```
env:
  GHCR_IMAGE_FQDN: ghcr.io/jeremylong/open-vulnerability-data-mirror
  HUB_IMAGE_FQDN: jeremylong/open-vulnerability-data-mirror
```

```
          tags: |
            ${{ env.GHCR_IMAGE_FQDN }}:${{ env.VERSION }}
            ${{ env.GHCR_IMAGE_FQDN }}:latest
            ${{ env.HUB_IMAGE_FQDN }}:${{ env.VERSION }}
            ${{ env.HUB_IMAGE_FQDN }}:latest
```

Hope this is fine too?

Reply:
good enough